staff-guidelines.md

```diff
@@ -32,3 +32,23 @@ Following social contract should be one of the considerations of “performance
   - Initiatives (e.g. Gamejams, TOPathons, Fireside chats)
   - Etc.
 - Communications should happen on a semi-regular cadence
+
+---
+
+# Procedure for reporting a concern with an Odin Staff Member.
+
+Odin staff members include all members of the Core, Maintainer and Moderators Teams. Concerns can include, but should not be limited to infractions against the Social Contract, team expectations, and Discord rule infractions.

```

Comment:
```suggestion
Odin staff members include all members of the Core, Maintainer and Moderator teams. Concerns include but are not  limited to infractions against the Social Contract, team expectations, and Discord rule infractions.

```
Minor grammar nits, I think this flows a little better

Reply:
[8270377](https://github.com/TheOdinProject/top-meta/pull/186/commits/82703773e8d004c679e3d5dbcd725589b7d1cce1)